src/extensions/LensExtension/LensExtension.js

```diff
@@ -85,13 +91,16 @@ const LensExtension = class extends LayerExtension {
   }
 
   draw() {
-    const { unprojectLensBounds } = this.state;
+    const { unprojectLensBounds = [0, 0, 0, 0] } = this.state;
     const {
       bounds,
-      isLensOn,
-      lensSelection,
-      lensBorderColor = [255, 255, 255],
-      lensBorderRadius = 0.02
+      isLensOn = defaultProps.isLensOn.value,
+      lensSelection = defaultProps.lensSelection.value,
+      lensBorderColor = defaultProps.lensBorderColor.value,
+      lensBorderRadius = defaultProps.lensBorderRadius.value,
+      colors,
```

Comment:
Maybe `lensColors` and then have a note that `colors` will be used if no `lensColors` is specified.

Reply:
I would say we should leave this one be.  I think it could be confusing with the other props here and this will signal that this does the same exact thing as the `ColorPaletteExtension`